src/Core/Form/IdentifiableObject/Builder/FormBuilderFactoryInterface.php

```diff
@@ -30,6 +30,8 @@
 
 /**
  * Defines interface for creating form builders.
+ *
+ * @todo in next major: Add ?FormOptionProviderInterface $optionProvider = null as a third parameter to this interface
```

Comment:
May be create an issue milestoned in PrestaShop 8.0 would be a better solution.

Reply:
I can do both, I'm not sure about the visibility that this issue will still have in a year from now So I prefer to leave the todo as well just in case (even if ti's not that much more visible ^^)

Reply:
I created this issue https://github.com/PrestaShop/PrestaShop/issues/24106 The 8.0 milestone doesn't exist So I used `Next` instead 🤷‍♂️

Reply:
I agree. We have way too many issues, which means finding some item is very hard. @ldeprestashop (Laurent) wants to reduce the number of issues rather than increase it ^^ .

What's more: we are not sure yet of the scope of the BC breaks expected for PS 8.0.0 .

It is possible, for example, that for PS 8.0.0 we say "in order to ease the migration from PS 1.7 to PS 8.0 we have only broken backward compatibility on the items on the list A, and nothing else". Depending on list A content we might not be able to deprecate every single "todo next major" 🤔  .

As this has to be discussed, I find @jolelievre quite interesting 👍